buffer/moon.pkg.json

```diff
@@ -1,4 +1,3 @@
 {
-  "import": ["moonbitlang/core/builtin"],
-  "test-import": ["moonbitlang/core/coverage"]
```

Comment:
I can't understand why coverage need to listed in "import" but not "test-import". @lynzrand

Reply:
My best guess is that we didn't have `test-import` at all at the time when coverage was added.

You can see from `hashset/moon.pkg.json`: When coverage was added https://github.com/moonbitlang/core/blob/b5f6010e093b40c88d6dee36510f2979adcfbbad/hashset/moon.pkg.json; When `test-import` was first added (it was snake-case instead of kebab-case at that time): https://github.com/moonbitlang/core/blob/dbf949f10f95802d69f23150d26fc2bcb5f4b6c1/hashset/moon.pkg.json

Reply:
/cc @Young-Flash @lijunchen